dashboard/app/models/user.rb

```diff
@@ -641,7 +641,6 @@ def self.find_channel_owner(encrypted_channel_id)
     nil
   end
 
-  validate :presence_of_email, if: -> {teacher? && purged_at.nil?}
```

Comment:
How are we ensuring teachers are created with an email now?

Reply:
There was already this:
```
  validate :presence_of_email_or_hashed_email, if: :email_required?, on: :create
```

Reply:
There is already this validation:
```
  validate :presence_of_email_or_hashed_email, if: :email_required?, on: :create
```
```
  def email_required?
    return true if teacher?
    return false if manual?
    return false if sponsored?
    return false if oauth?
    return false if parent_managed_account?
    true
  end
```

However, now that you have me look at it again. I guess its still possible for a teacher to be created with just a hashed_email and still missing the email.

I will fix this.